From 51a8885683a2f07ed8fc79741947a47fccdf4fce Mon Sep 17 00:00:00 2001
From: Chao Jiang <chao.jiang@manchester.ac.uk>
Date: Wed, 26 Jul 2017 14:44:34 +0100
Subject: [PATCH] VIM: fix font config

---
 vim/vimrc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/vim/vimrc b/vim/vimrc
index 465e9d1..57d05c9 100644
--- a/vim/vimrc
+++ b/vim/vimrc
@@ -112,7 +112,7 @@ set exrc
 set secure
 set hlsearch
 set nu
-set guifont=Andale\ Mono:h11
+set guifont=Andale\ Mono\ 10
 set guioptions-=T	"remove toolbar
 
 " key remap
-- 
2.13.3

