From 3fdf8bd082275ae751eb844d1d920b7d609cf4bd Mon Sep 17 00:00:00 2001
From: Chao Jiang <changjinglu2@hotmail.com>
Date: Thu, 27 Jul 2017 05:42:32 +0100
Subject: [PATCH 2/2] VIM: plugin selection for windows

---
 vim/vimrc | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/vim/vimrc b/vim/vimrc
index 4be6a99..0cf4f7d 100644
--- a/vim/vimrc
+++ b/vim/vimrc
@@ -30,9 +30,9 @@ Plugin 'cyanjc321/Smart-Tabs'
 Plugin 'scrooloose/nerdtree'
 Plugin 'Xuyuanp/nerdtree-git-plugin'
 " Plugin 'scrooloose/syntastic'
-Plugin 'Valloric/YouCompleteMe'
-Plugin 'jeaye/color_coded'
-Plugin 'rdnetto/YCM-Generator'
+" Plugin 'Valloric/YouCompleteMe'
+" Plugin 'jeaye/color_coded'
+" Plugin 'rdnetto/YCM-Generator'
 " Plugin 'taglist.vim'
 Plugin 'majutsushi/tagbar'
 Bundle 'craigemery/vim-autotag'
@@ -43,7 +43,7 @@ Plugin 'honza/vim-snippets'
 Plugin 'tpope/vim-fugitive'
 Plugin 'euclio/vim-nocturne'
 Plugin 'flazz/vim-colorschemes'
-Plugin 'christoomey/vim-tmux-navigator'
+" Plugin 'christoomey/vim-tmux-navigator'
 Plugin 'vim-airline/vim-airline'
 Plugin 'vim-airline/vim-airline-themes'
 Plugin 'CSApprox'
-- 
2.13.2.windows.1

