From e2c6647d070ac44c98ead9ced7cf1d5d57eca270 Mon Sep 17 00:00:00 2001
From: Chao Jiang <chao.jiang@manchester.ac.uk>
Date: Tue, 16 Jan 2018 18:28:08 +0000
Subject: [PATCH 2/2] LINUX: VIM: disabled color_coded due to compatibility
 issue

---
 vim/vimrc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/vim/vimrc b/vim/vimrc
index 4e93227..04ca5eb 100644
--- a/vim/vimrc
+++ b/vim/vimrc
@@ -31,7 +31,7 @@ Plugin 'scrooloose/nerdtree'
 Plugin 'Xuyuanp/nerdtree-git-plugin'
 " Plugin 'scrooloose/syntastic'
 Plugin 'Valloric/YouCompleteMe'
-Plugin 'jeaye/color_coded'
+" Plugin 'jeaye/color_coded'
 Plugin 'rdnetto/YCM-Generator'
 " Plugin 'taglist.vim'
 Plugin 'majutsushi/tagbar'
-- 
2.15.1

