From 3e8af3c40aaa43bd2d31bf21ac6d45c15a5d1a23 Mon Sep 17 00:00:00 2001
From: Chao Jiang <chao.jiang@manchester.ac.uk>
Date: Wed, 26 Jul 2017 14:49:59 +0100
Subject: [PATCH] TMUX: xclip for copy to clipboard in linux

---
 tmux/tmux.conf | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tmux/tmux.conf b/tmux/tmux.conf
index c40058a..3350ba8 100644
--- a/tmux/tmux.conf
+++ b/tmux/tmux.conf
@@ -38,11 +38,11 @@ setw -g mode-keys vi
 
 # setup 'v' to begin selection as in vim
 bind-key -T copy-mode-vi v send-keys -X begin-selection
-bind-key -T copy-mode-vi y send-keys -X copy-pipe "reattach-to-user-namespace pbcopy"
+bind-key -T copy-mode-vi y send-keys -X copy-pipe "xclip -in -selection clipboard"
 
 # update default binding of `Enter` to also use copy-pipe
 unbind-key -T copy-mode-vi Enter
-bind-key -T copy-mode-vi Enter send-keys -X copy-pipe "reattach-to-user-namespace pbcopy"
+bind-key -T copy-mode-vi Enter send-keys -X copy-pipe "xclip -in -selection clipboard"
 
 ######################
 ### DESIGN CHANGES ###
-- 
2.13.3

