From 0fce37469620c0a39e5e96abda820421311fb22f Mon Sep 17 00:00:00 2001
From: Chao Jiang <chao.jiang@manchester.ac.uk>
Date: Wed, 26 Jul 2017 14:59:57 +0100
Subject: [PATCH 1/2] ZSH: fixed file locations

---
 zsh/zshrc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/zsh/zshrc b/zsh/zshrc
index 1b53d94..9f92888 100644
--- a/zsh/zshrc
+++ b/zsh/zshrc
@@ -5,13 +5,13 @@ SAVEHIST=1000
 bindkey -v
 # End of lines configured by zsh-newuser-install
 # The following lines were added by compinstall
-zstyle :compinstall filename '/Users/jiangchao/.zshrc'
+zstyle :compinstall filename '/home/cjiang/.zshrc'
 
 autoload -Uz compinit
 compinit
 # End of lines added by compinstall
 # Lines for antigen setup
-source $(brew --prefix)/share/antigen/antigen.zsh
+source /usr/share/zsh/share/antigen.zsh
   
 antigen use oh-my-zsh
 antigen theme robbyrussell
-- 
2.13.3

